Jenkinsfile

```diff
@@ -38,7 +38,7 @@ pipeline {
           sh "rm -rf {.*,*} || ls -la && git clone https://\$GIT_USER:\$GIT_PASSWORD@github.com/dcos/dcos-ui.git ."
         }
         sh "git fetch"
-        sh "git checkout \"\$([ -z \"\$CHANGE_TARGET\" ] && echo \$BRANCH_NAME || echo \$CHANGE_TARGET )\""
+        sh "git checkout \"\$([ -z \"\$CHANGE_BRANCH\" ] && echo \$BRANCH_NAME || echo \$CHANGE_BRANCH )\""
```

Comment:
`git merge` is missing 🤔

Reply:
That was missing before, can we make it a not-emergency task to add it? We would have been living without it a month anyways. I would feel better if I knew we had the time to do this properly.

Reply:
please create a follow-up PR. I'd say not having merge is worse than not linting commits…

Reply:
actually… We're enforcing that PRs are up to date which ends up in us not having them to be merged with master 🤔